src/components/Table/utils/styles.tsx

```diff
@@ -0,0 +1,182 @@
+import { ReactNode } from "react";
+import { PresentationContextProps, PresentationFieldProps } from "src/components/PresentationContext";
+import { DiscriminateUnion, GridColumn, GridDataRow, GridTableApi, RenderCellFn } from "src/components/Table";
+import { Kinded, RenderAs } from "src/components/Table/types";
+import { Css, Palette, Properties } from "src/Css";
+import { safeKeys } from "src/utils";
+
+/** Completely static look & feel, i.e. nothing that is based on row kinds/content. */
+export interface GridStyle {
```

Comment:
Thought: Fwiw I think I'd put this in a `GridStyle.ts` right next to `GridTable.ts`; can't quite articulate, just that it seems more important / part of the core public API and not just a utils things.